### ISSUE TITLE
Explicit definition of botocore client credentials considered harmful

### ISSUE THREAD

**Issue by manualautomaton** [withtwoemms/recruitment#25]
[this change](https://github.com/withtwoemms/recruitment/commit/1b67a99663c562ef03c2c6b2e0933fc0d35c9709#diff-a12cf9103573289bc602f332520cbed19cb98b64a92a2ba7c15f68828a5c9c33R115) seems to lead to this failure:
```
. . .
File "/Users/emmanuel.obi/.pyenv/versions/newsela-cli/lib/python3.10/site-packages/botocore/client.py", line 415, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/Users/emmanuel.obi/.pyenv/versions/newsela-cli/lib/python3.10/site-packages/botocore/client.py", line 746, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.errorfactory.UnrecognizedClientException: An error occurred (UnrecognizedClientException) when calling the GetLogEvents operation: The security token included in the request is invalid.
```

**Comment by manualautomaton** [comment]
v0.4.1a1 works as expected

**Comment by withtwoemms** [comment]
resolved via release of v0.4.1
